README.md

```diff
@@ -37,6 +37,19 @@ fail. For instructions on deploying MLFlow you can start with [this
 guide](https://discourse.charmhub.io/t/deploying-charmed-mlflow-v2-and-kubeflow-to-eks/10973),
 ignoring the EKS specific steps.
 
+## Python and Tox Setup
+To run the tests, Python 3.8 and Tox must be installed on your system. If your default Python version is higher than 3.8, you can set up Python 3.8 and Tox with the following commands:
+
+```bash
+sudo add-apt-repository ppa:deadsnakes/ppa -y
+sudo apt update -y
+sudo apt install python3.8 python3.8-distutils python3.8-venv -y
+wget https://bootstrap.pypa.io/get-pip.py
```

Comment:
I am okay with these commands up until this `wget` command.

The thing is that we are telling people to run the following (see [here](https://github.com/canonical/charmed-kubeflow-uats/blob/82ef13a76b8015fe2d2f00ab5f5c71d87c6c15e0/README.md#running-from-a-configured-management-environment-using-the-driver)):

```
python3 -m venv venv <--- here they will create a venv
source venv/bin/activate
pip install tox <--- here they will install tox
```

so what I think this guide should say is that tests will run from inside a venv, which has to have py3.8. They can then proceed to activate the venv and install tox from it. wdyt?

Reply:
Adjusted the text :) wdyt?